src/main/java/wedlog/address/logic/commands/VendorAddCommand.java

```diff
@@ -1,25 +1,83 @@
 package wedlog.address.logic.commands;
 
+import static java.util.Objects.requireNonNull;
+import static wedlog.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static wedlog.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static wedlog.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static wedlog.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static wedlog.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import wedlog.address.commons.util.ToStringBuilder;
+import wedlog.address.logic.Messages;
 import wedlog.address.logic.commands.exceptions.CommandException;
 import wedlog.address.model.Model;
 import wedlog.address.model.person.Vendor;
 
 /**
- * Adds a Vendor to the address book.
+ * Adds a Vendor to Wedlog.
  */
 public class VendorAddCommand extends Command {
     // implementation more or less copied from AddCommand
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a vendor to the address book. ";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a vendor to the address book. "
+            + "Compulsory Parameters: "
+            + PREFIX_NAME + "NAME "
+            + "Optional Parameters: "
+            + PREFIX_PHONE + "PHONE "
+            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_TAG + "florist";;
```

Comment:
I believe the message is missing a vendor identifier. Perhaps you can add constant `VendorCommandParser.COMMAND_WORD` so that we do not have to write "vendor" here and there?

```suggestion
    public static final String MESSAGE_USAGE = VendorCommandParser.COMMAND_WORD + " " 
            + COMMAND_WORD + ": Adds a vendor to the address book. "
            + "Compulsory Parameters: "
            + PREFIX_NAME + "NAME "
            + "Optional Parameters: "
            + PREFIX_PHONE + "PHONE "
            + PREFIX_EMAIL + "EMAIL "
            + PREFIX_ADDRESS + "ADDRESS "
            + "[" + PREFIX_TAG + "TAG]...\n"
            + "Example: " + COMMAND_WORD + " "
            + PREFIX_NAME + "John Doe "
            + PREFIX_PHONE + "98765432 "
            + PREFIX_EMAIL + "johnd@example.com "
            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
            + PREFIX_TAG + "florist";
```

Reply:
I couldn't find a VendorCommandParser.COMMAND_WORD on my end. Plus, we already have the line "Adds a vendor to the address book." Can I clarify what the inclusion of VendorCommandParser.COMMAND_WORD is supposed to add on top of that?

Reply:
Currently, the message starts with `add` only, i think it would be neater if it started with `vendor add`.

I forgot to include this in the suggestion but the more important missing `vendor` is in the example. Line 32 should be `+ "Example: vendor" + COMMAND_WORD + " "`.

To clarify the point on `VendorCommandParser.COMMAND_WORD` (`"vendor"`), it currently doesn't exist. I sugggest by introducing such a constant, we reduce the occurence of [magic numbers](https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers), such as in the switch case of `AddressBookParser`.